src/Raven.Server/Documents/Indexes/IndexStore.cs

```diff
@@ -1265,7 +1265,7 @@ internal IndexCreationOptions GetIndexCreationOptions(object indexDefinition, In
             if ((differences & IndexDefinitionCompareDifferences.DeploymentMode) == IndexDefinitionCompareDifferences.DeploymentMode)
                 return IndexCreationOptions.UpdateWithoutUpdatingCompiledIndex;
 
-            return IndexCreationOptions.Update;
+            return IndexCreationOptions.UpdateWithoutUpdatingCompiledIndex;
```

Comment:
1. There is no test
2. This is not acceptable, this is the last case, if you are here and you do not know what is going on, you need to update everything

Reply:
The problem is that we may need to update the configuration of an existing index with values that would make sense in the next value.